### DISCUSSION TITLE
Welcome to filament-settings Discussions!

### DISCUSSION THREAD

**Discussion by quintenbuis** [reworck/filament-settings#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jonykpi** [comment]
Hey, its issue with put a file, its give direct error,
error https://prnt.sc/RckSusD-yied
view https://prnt.sc/HnUYFEJ0HZvT
i am just using this
 FileUpload::make('_logo')->image()
 is there any solution ?

**Reply by netzperfekt** [reply]
having the same issue - can you help?

**Reply by netzperfekt** [reply]
found the problem: pages/settings.php#42 should be:

`foreach ($this->form->getState() as $key => $data) {`

(not ...$this->data...) because in getstate() among other things the upload fields are moved correctly. with this modification it's working without problems
